src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy

```diff
@@ -94,6 +97,26 @@ class K8sServiceImpl implements K8sService {
     @Nullable
     private String requestsMemory
 
+    @Value('${wave.build.k8s.toleration.arm64.enabled:false}')
+    @Nullable
+    private boolean tolerationARM64Enabled
+
+    @Value('${wave.build.k8s.toleration.arm64.key}')
+    @Nullable
+    private String tolerationARM64Key
+
+    @Value('${wave.build.k8s.toleration.arm64.value}')
+    @Nullable
+    private String tolerationARM64Value
+
+    @Value('${wave.build.k8s.toleration.arm64.operator:`Equal`}')
+    @Nullable
+    private String tolerationARM64Operator
+
+    @Value('${wave.build.k8s.toleration.arm64.effect:`NoSchedule`}')
```

Comment:
```suggestion
    @Value('${wave.build.k8s.toleration.arm64.effect}')
```

The effect can't have a default, since the docs say that an empty effect parameter is allowed:
> An empty effect matches all effects with key key1.

Reply:
Empty is valid for k8s, but we need to think about our use case and what can be a default scenario.
In this case, we want to schedule arm pods in arm nodes.

Reply:
> but we need to think about our use case and what can be a default scenario

Sorry, I think we need to think what customers may want to use, instead of hardcoding our deployment usecase into the wave config.